src/modules/numberToWords/numbersWordList.ts

```diff
@@ -0,0 +1,49 @@
+const numbersWordList: Map<number, string> = new Map([
```

Comment:
Please check the browser compatibility after adding this function to ensure that the bundle works well in old nodejs and browsers version.